README.md

```diff
@@ -1,6 +1,8 @@
 Torrentor!
 =============
 
+![Torrentor app](torrentor.png "Torrentor")
```

Comment:
The screenshot would look beter after the introductory sentence.

Reply:
Changed it.